src/graphql.service.ts

```diff
@@ -5,136 +5,157 @@ import {
                 local: '',
             },
         },
         // Boba ETH
         288: {
             [EGraphQLService.AnchorageBridge]: {
-                gql: "https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/anchorage-bridge-boba-eth/v1/gn"
+                gql: this.withSubgraphId("BkDvK8QHM8HRHCQtM6CSBjF4ipH1rW4uLBnMGN28JzwM"),
             },
             [EGraphQLService.DAO]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/dao-boba-eth/v1/gn',
+                gql: this.withSubgraphId('2p7KCVnBeNSnTVwMfJC9NENtEKwZn6hv45CU4Cf6PczW'),
                 local: '',
             },
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-boba-eth/v1/gn',
+                gql: this.withSubgraphId('34UUDYdzZFX7afhysqX7JodzKngJmjsFKJDtWsTxi9UA'),
                 local: '',
             },
         },
         // Boba BNB
         56288: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-boba-bnb/v1/gn',
+                gql: 'https://graphql.bnb.boba.network/subgraphs/name/boba-bnb-l2/',
                 local: '',
             },
+            [EGraphQLService.AnchorageBridge]: {
+                gql: 'https://graphql.bnb.boba.network/subgraphs/name/anchorage-bridging-boba-bnb',
+            },
         },
         // BSC
         56: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-bsc/v1/gn',
+                gql: this.withSubgraphId('4Eoi62aX8V1DCebNX4mzPi7rfceFaRAeZ3xt7AYVowpy'),
                 local: '',
             },
         },
         // BNB testnet
         97: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-chapel/v1/gn',
+                gql: this.withSubgraphId('B8nR7fCE2KFV3oMJcLTDd56p7WQsoPH7wvBRRoqmocP8'),
                 local: '',
             },
+            [EGraphQLService.AnchorageBridge]: {
+                gql: this.withSubgraphId('7bHCzdBfrkEHegRUwKJEkNG5moo6Yv6NmYdA9rQBCufy'),
+            },
         },
         // Boba BNB testnet
         9728: {
             [EGraphQLService.LightBridge]: {
-                gql: 'https://api.goldsky.com/api/public/project_clq6jph4q9t2p01uja7p1f0c3/subgraphs/light-bridge-boba-bnb-testnet/v1/gn',
+                gql: 'https://graphql.testnet.bnb.boba.network/subgraphs/name/boba-bnb-l2-testnet/',
                 local: 'http://127.0.0.1:8002/subgraphs/name/boba/Bridges',
             },
+            [EGraphQLService.AnchorageBridge]: {
+                gql: 'https://graphql.bnb.boba.network/subgraphs/name/anchorage-bridging-boba-bnb',
+            },
```

Comment:
@wsdt looks like subgraphs is refered as mainnet one like Boba BNB above.

https://github.com/bobanetwork/graphql-utils/pull/6/files#diff-598c468d2a6aebfb01813d5057757685113469de2752fb80cadc288859448389R73-R75

Reply:
@sk-enya what do you mean with is referred as mainnet? It is within the boba bnb mainnet config

Reply:
I mean this url is for BOBA bnb mainnet subgraph.!